design/pixel-art/index.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+---
+# pixel-art
+
```

Comment:
Idee: noch klingt es sehr sachlich und wenig nach "Verkauf"...
Vorschlag: "Vielleicht könnt ihr euch noch an die Zeiten erinnern, in denen ihr selber als Kinder Spaß hattet mit Games auf der Playstation___ oder auf dem ersten Gameboy. Heute sind sie vielleicht technisch überholt, aber es gibt sie immer noch, die guten alten  Pixelgrafiken - und erleben unter Kultstatus ihr Comeback. Selber-anmalen.de zeigt, wie ihr die coolen alten Zeiten wieder aufleben lassen könnt. Hier findet ihr kostenlose Vorlagen zur Gestaltung eurer Lieblings-Pixelgrafiken - entweder auf Papier oder auf Textil!

Reply:
Erledigt - danke.